.gitlab-ci.yml

```diff
@@ -5,11 +5,15 @@ stages:
   - single-step-instrumentation-tests
   - macrobenchmarks
 
+check_config_inversion_local_file:
+  rules:
+    - when: on_success
+  variables:
+    LOCAL_JSON_PATH: "packages/dd-trace/src/supported-configurations.json"
+
 include:
   # See: https://gitlab.ddbuild.io/DataDog/apm-reliability/libdatadog-build
-  - project: DataDog/apm-reliability/libdatadog-build
-    ref: 0f677257308e1c379af490b754febfb40fa2c06d
-    file: templates/one-pipeline.yml
+  - remote: https://gitlab-templates.ddbuild.io/libdatadog/include-wip/anna/config-inversion-add-step/one-pipeline.yml
```

Comment:
The CI pipeline PR has to be merged before and this has to be updated before landing